crates/streamer/src/streamer/mod.rs

```diff
@@ -213,7 +281,7 @@ fn spawn_sending_task<W>(
         let task_result = task.await;
 
         if let Err(e) = task_result {
-            tracing::error!("Error while sending data: {:?}", e);
+            tracing::warn!(error=?e);
```

Comment:
note: For anyhow::Error, you should use the alternate formatting of Display. `error = format!("{e:#}")`. If it’s not anyhow::Error, just use the normal Display `error = %e`.

Reply:
updated

Reply:
I think I got them all, I made every `?e` into `error=%e` if display implemented, otherwise "error=?e", and I made anyhow errors to "error = format!("{e:#}")". Also made the error message start with lowercase letters for error and warn, used uppercase letters for info! and debug!

Reply:
Great, thank you!